run_dir/design/project_samples.html

```diff
@@ -154,13 +154,13 @@ <h4>User project description
         <div class="col-sm-4">
           <table class="table table-bordered table-striped">
             <thead>
-              <tr><th colspan="2">Bioinformatics</th></tr>
+              <tr><th colspan="2">Bioinformatics <a id="statusdb_link" data-toggle="tooltip" title="Opens project page in StatusDB" target="_blank">&#x1f517;</a>
+              </th></tr>
             </thead>
             <tbody>
-              <tr><td>Uppnex ID</td><td><code id="uppnex_id"></code></td></tr>
+              <tr><td>Delivery Type</td><td><code id="uppnex_id"></code></td></tr>
```

Comment:
Is it worth updating the `id` as well? It can be confusing if the code is still referring to `uppnex_id` when the page labels it as delivery type..

This is only if it's not going to add a lot of work.